program/src/lib.rs

```diff
@@ -7,9 +7,6 @@ pub mod instruction;
 pub mod processor;
 pub mod state;
 
-// Export current SDK types for downstream users building with a different SDK
-// version
-pub use solana_program;
```

Comment:
Given the removal of `solana-program`, this does not feel relevant anymore (?)

Reply:
Yeah feel free to remove